src/createServer.js

```diff
@@ -1,3 +1,72 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+
+const typeOfCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const messages = {
+  putText: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  insertQueryParam: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  notCorrectCase: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+const PORT = process.env.PORT || 4000;
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const newUrl = new URL(req.url, `http://localhost:${PORT}`);
+    const originalText = newUrl.pathname.slice(1);
+    const targetCase = newUrl.searchParams.get('toCase');
+    const errorsArray = [];
+
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
```

Comment:
You can't be sure that the request will succeed without errors. It's better to assign success code after all checks have passed

Reply:
> You can't be sure that the request will succeed without errors. It's better to assign success code after all checks have passed

Good point